pkg/cgroups/cgroups_linux.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cgroups
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cilium/cilium/pkg/mountinfo"
+	"golang.org/x/sys/unix"
```

Comment:
Nit: we usually separate import groups for cilium/cilium and 3rd party packages, i.e.

```suggestion
	"github.com/cilium/cilium/pkg/mountinfo"
	
	"golang.org/x/sys/unix"
```